lib/internal/streams/from.js

```diff
@@ -4,13 +4,25 @@ const {
   Symbol,
   SymbolIterator
 } = primordials;
+const { Buffer } = require('buffer');
 
 const {
   ERR_INVALID_ARG_TYPE
 } = require('internal/errors').codes;
 
 function from(Readable, iterable, opts) {
   let iterator;
+  if (typeof iterable === 'string' || iterable instanceof Buffer) {
+    return new Readable({
+      objectMode: true,
```

Comment:
Not sure of the purpose here, but wouldn't `objectMode: false` make more sense here as a default? Or is that breaking?

Reply:
I would keep it consistent with the rest. Also, it would change the encoding, so possibly it's not a good idea.

Reply:
The problem with objectMode though is that `setEncoding` doesn't do anything? Which might be expected of a buffer/string steam?